splash/browser_tab.py

```diff
@@ -95,6 +100,53 @@ def _init_webpage(self, verbosity, network_manager, splash_proxy_factory,
         self.web_page.setNetworkAccessManager(network_manager)
         self.web_page.splash_proxy_factory = splash_proxy_factory
         self.web_page.render_options = render_options
+        
+        
+        def unsupported_content_handler(reply):
```

Comment:
could you please make it a method? You can pass different arguments using e.g. functools.partial